py/server/deephaven/updateby.py

```diff
@@ -483,7 +490,13 @@ def forward_fill(cols: Union[str, List[str]]) -> UpdateByOperation:
 
 
 def delta(cols: Union[str, List[str]], delta_control: DeltaControl = DeltaControl.VALUE_DOMINATES) -> UpdateByOperation:
-    """Creates a delta UpdateByOperation for the supplied column names.
+    """Creates a delta UpdateByOperation for the supplied column names. The Delta operation produces values by computing
+    the difference between the current value and the previous value. When the current value is null, this operation
+    will output null. When the current value is valid, the output will depend on the DeltaControl provided.
+
+        When delta_control is not provided or set to VALUE_DOMINATES, a valid value following a null value returns the
+            valid value.
+        When delta_control is set to NULL_DOMINATES, a valid value following a null value returns null.
```

Comment:
Now that we have some docs, I have a much better understanding of what is going on.  Here is my understanding:

1) `VALUE_DOMINATES` treats the prior null value as 0, so the current value is returned as the current delta.
2) `NULL_DOMINATES` poisons the result and results in `null` for the current delta.

If this interpretation is correct, I'm thinking about the default value and which option produces the least surprise.  I might argue that `NULL_DOMINATES` would be less surprising, but we should get feedback from @jmao-denver @jjbrosnan @alexpeters1208 .

Reply:
You interpret correctly.  This was a port from DHE and I simply propagated the default of `VALUE_DOMINATES` without introspection.